src/librawspeed/tiff/TiffEntry.h

```diff
@@ -78,7 +78,7 @@ class TiffEntry
   ushort16 getShort(uint32 num=0) const;
   short16 getSShort(uint32 num=0) const;
   float getFloat(uint32 num=0) const;
-  std::string getString() const;
+  std::string getString(bool trimmed = true) const;
```

Comment:
In any case, `getId()` _needs_ to keep calling trim.
It is called from `addCamera()`, which could be called not by rawspeed, and unless the data gets the same treatment (read: trimming), the results may be unexpected.